commands/list-selectors.js

```diff
@@ -1,18 +1,39 @@
 // eslint-disable-next-line no-unused-vars
 var colors = require('colors'),
-  fs = require('fs')
+  fs = require('fs'),
+  _ = require('lodash')
 
 module.exports = function (program) {
   program
     .command('list-selectors')
     .description('list available selectors')
-    .action(function (/*cmd*/) {
+    .option('-e, --exchange <exchange>', 'Filter to only show given exchange (default: '*')',String,'*')
+    .option('-c, --currency <currency>', 'Filter to only show given currency (default: '*')',String,'*')
+    .option('-a, --asset <asset>', 'Filter to only show given asset (default: '*')',String,'*')
+    .option('-d, --no-details <details>', 'Prints selectors without additional details')
+    .action(function(cmd) {
       var exchanges = fs.readdirSync('./extensions/exchanges')
+      if (cmd.exchange != '*') {
+        exchanges = _.filter(exchanges,(item)=>{
+          return cmd.exchange === item
+        })
+      }
       exchanges.forEach(function(exchange){
         if (exchange === 'sim' || exchange === '_stub') return
 
-        console.log(`${exchange}:`)
+        if (cmd.details === 'true') console.log(`${exchange}:`)
```

Comment:
shouldn't this be `=== false`? or is it specifically `'true'` when not set?

Reply:
`!cmd.details` :thinking: